templates/autoscale-existing-vpc.template.yaml

```diff
@@ -743,6 +752,18 @@ Metadata:
                 default: Custom asset S3 bucket
             CustomAssetDirectory:
                 default: Custom asset folder
+            FortiAnalyzerIntegrationOptions:
+                default: FortiAnalyzer integration
+            FortiAnalyzerVersion:
+                default: FortiAnalyzer version
+            FortiAnalyzerInstanceType:
+                default: FortiAnalyzer instance type
+            FortiAnalyzerCustomPrivateIpAddress:
+                default: FortiAnalyzer custom IP
+            FortiAnalyzerAutoscaleAdminUsername:
+                default: Autoscale admin username
+            FortiAnalyzerAutoscaleAdminPassword:
+                default: Autoscale admin password
```

Comment:
Hi @klc178 , could you please review the labels for accuracy?

Reply:
I patched the change so they are now consistent with what we have in the AWS QSG project.
see the patch here: https://github.com/fortinet/fortigate-autoscale-aws/pull/29/commits/e0cca155fb6b8dec042ee1c3bcb283625f7bb1a0